### PR TITLE
bypass: really bypass udp flow from first packet

### PR DESCRIPTION
**Pull request by catenacyber**
Link to ticket: https://redmine.openinfosecfoundation.org/issues/
https://redmine.openinfosecfoundation.org/issues/7053

Describe changes:
- bypass: really bypass udp flow from first packet

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1870

### CODE REVIEW

**Comment by catenacyber** [comment]
SV test would show
    flow.state as "bypassed"
and stats having
          "local_bypassed": 1

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 20876

File: src/flow.c [hunk 1]
```diff
@@ -485,7 +485,13 @@ void FlowHandlePacketUpdate(Flow *f, Packet *p, ThreadVars *tv, DecodeThreadVars
         SCLogDebug("pkt %p FLOW_PKT_ESTABLISHED", p);
         p->flowflags |= FLOW_PKT_ESTABLISHED;
 
-        FlowUpdateState(f, FLOW_STATE_ESTABLISHED);
+        if (
+#ifdef CAPTURE_OFFLOAD
+                (f->flow_state != FLOW_STATE_CAPTURE_BYPASSED) &&
+#endif
+                (f->flow_state != FLOW_STATE_LOCAL_BYPASSED)) {
+            FlowUpdateState(f, FLOW_STATE_ESTABLISHED);
```
Comment (victorjulien) [inline src/flow.c:493]: should we flag the packet as part of an established flow in this case, as we do on line 486?

Comment (catenacyber) [reply src/flow.c:493]: I guess so.

We have
1. UDP from client to server, nothing special
2. UDP from server to client, we trigger a `bypass` rule that `FlowUpdateState(FLOW_STATE_LOCAL_BYPASSED)`, and in later processing, we call `FlowHandlePacketUpdate` that also sees the flow as established...

**Comment by catenacyber** [comment]
Rebased in https://github.com/OISF/suricata/pull/11287

**Pull request closed without merging**